gcp/appengine/frontend_handlers.py

```diff
@@ -77,6 +79,54 @@ def index():
   return render_template('index.html')
 
 
+@blueprint.route('/v2/')
+def index_v2():
+  return render_template('home.html')
+
+
+@blueprint.route('/v2/list')
+def list():
+  """Main page."""
+  # TODO: Can/should we do this as an internal query not a full on request to osv.dev?
+  page = request.args.get('page') if request.args.get('page') and request.args.get('page').isnumeric() else '1'
+  response = requests.get('https://osv.dev/backend/query?page=%s&search=&affected_only=true&ecosystem=' % page)
+  results = json.loads(response.content)
+
+  vulnerabilities = []
+  for item in results['items']:
+    vulnerabilities.append({
+      "id": item['id'],
+      "summary": item['summary'] if 'summary' in item else '',
+      "packages": item['affected'][0]['package']['ecosystem'],
+      "versions": item['affected'][0]['versions']
+    })
+
+  return render_template('list.html', vulnerabilities = vulnerabilities)
+
+
+@blueprint.route('/v2/vulnerability/<id>')
+def vulnerability(id):
+  """Vulnerability page."""
+  # TODO: Can/should we do this as an internal query not a full on request to osv.dev?
+  response = requests.get('https://osv.dev/backend/vulnerability?id=%s' % id)
+  item = json.loads(response.content)
+
+  vulnerability = {
+    "id": item['id'],
+    "summary": item['summary'] if 'summary' in item else '',
+    "packages": item['affected'][0]['package']['ecosystem'],
```

Comment:
This should be item['affected'][0]['package'] (not only taking the "ecosystem"), and we'd want to render it something like "{ecosystem}/{name}". e.g. "PyPI/django"

It's fine for now to only show the [0] item, but we'd also want to show every single entry in the `affected` entry in the final version.

Reply:
Yep this was a temporary hack to get a WIP table going, let's get the table looking right in a followup PR